### ISSUE TITLE
Warn in the console that a deployment config is waiting on an image

### ISSUE THREAD

**Issue by kargakis** [openshift/origin#9041]
Forked from https://github.com/openshift/origin/issues/9018#issuecomment-221547070
> It might be nice to know in web console that the deployment is waiting for something. Currently it says No events to show.

We do this in `oc status` but not in the console.

cc: @fabianofranz @spadgett @tnozicka

**Comment by fabianofranz** [comment]
+1. I think I've hit this in Hack Day and wasn't yet sure how to reproduce.

**Comment by kargakis** [comment]
FWIW the way we determine how a config is waiting for an image in the controller is by looking at LastTriggeredImage in the ImageChange trigger. If empty, then we need to wait.

**Comment by kargakis** [comment]
Closing in favor of https://github.com/openshift/origin-web-console/issues/490
